ps_spec.yml

```diff
@@ -12,6 +12,11 @@ stages:
         compute_size: small
         commands:
             - make test
+            - git clone git@algithub.pd.alertlogic.net:alertlogic/al-ps-tools.git
+            - cp coverage/cobertura-coverage.xml al-collector-js.coverage.xml
+            - bash ./al-ps-tools/helpers/collect.sh -c 'al-collector-js.coverage.xml'
```

Comment:
let's move it to `Master Push - Publish` build phase.
Basically the idea is to publish coverage report per release, not per PR or test run